docs/src/main/sphinx/sql/alter-table.rst

```diff
@@ -82,9 +98,19 @@ Allow everyone with role public to drop and alter table ``people``::
 
     ALTER TABLE people SET AUTHORIZATION ROLE PUBLIC
```

Comment:
The interaction between pure SQL role granting and BIAC role granting is ... unexplored.

Reply:
Understood, but ultimately out of scope for this PR

Reply:
@Ordinant Do you mean using things like file-based access control together with SQL based access control?

If so such a configuration can never be possible since Trino only allows a single type of system access control to be available at a time.

However connectors may provide their own - and the rule there is that system level configuration wins over connector specific ones.